template/{{.input_root_dir}}/_params_testing_only.txt.tmpl

```diff
@@ -0,0 +1,31 @@
+input_root_dir={{.input_root_dir}}
```

Comment:
Is this file to test if all input params are populate?

Reply:
All input params are populated as the validation covers all combinations and compares cookiecutter vs templates.
This file is used by unit tests and it exists before the migration.